app/services/state_file/state_information_service.rb

```diff
@@ -30,11 +51,32 @@ def state_code_from_intake_class(klass)
     STATES_INFO = {
       az: {
         intake_class: StateFileAzIntake,
-        name: "Arizona",
+        state_name: "Arizona",
+        refund_url: "https://aztaxes.gov/home/checkrefund",
+        tax_payment_url: "AZTaxes.gov",
+        download_form_name: "Form AZ-140V",
+        mail_voucher_address: "Arizona Department of Revenue<br/>"\
+          "PO Box 29085<br/>"\
+          "Phoenix, AZ 85038-9085".html_safe,
+        pay_mail_online_link: "https://azdor.gov/making-payments-late-payments-and-filing-extensions",
+        vita_link: "https://airtable.com/appnKuyQXMMCPSvVw/pag0hcyC6juDxamHo/form",
+        voucher_path: "/pdfs/AZ-140V.pdf",
+        survey_link: "https://codeforamerica.co1.qualtrics.com/jfe/form/SV_7UTycCvS3UEokey",
```

Comment:
should i `.freeze` all these strings...?

Reply:
That feels verbose and like something that will come up repeatedly. I wonder if we should implement some kind of `deep_freeze` method for cases like this? (I'm surprised ruby doesn't have one built in).

Reply:
good point about it being repetitive. we actually do have a gem we're using in one place called `ice_nine` that has a `deep_freeze`